### PR TITLE
Log node errors

### PR DESCRIPTION
**Pull request by coyotte508**
For https://github.com/huggingface/transformers/pull/15710

### CODE REVIEW

**Review by mishig25 — APPROVED** [review]
lgtm! I think the output from node env is useful

File: src/doc_builder/commands/build.py [hunk 1]
```diff
@@ -130,7 +130,7 @@ def build_command(args):
             print("Installing node dependencies")
             subprocess.run(
                 ["npm", "ci"],
-                stderr=subprocess.PIPE,
+                # stderr=subprocess.PIPE,
```
Comment (sgugger) [inline src/doc_builder/commands/build.py:133]: If it's not used, why leave it commented out? Let's please clean up and remove it entirely.

File: src/doc_builder/commands/build.py [hunk 2]
```diff
@@ -144,7 +144,7 @@ def build_command(args):
             print("Building HTML files. This will take a while :-)")
             subprocess.run(
                 ["npm", "run", "build"],
-                stderr=subprocess.PIPE,
+                # stderr=subprocess.PIPE,
```
Comment (sgugger) [inline src/doc_builder/commands/build.py:147]: Same here.

File: src/doc_builder/commands/build.py (hunk 1 above)
Comment (coyotte508) [reply src/doc_builder/commands/build.py:133]: Done in 6adc10b5ede2281995ff04d95f3c94db2bbebc96

Comment (sgugger) [reply src/doc_builder/commands/build.py:133]: Thanks!

**Pull request merged**